server/src/api/boards.go

```diff
@@ -60,6 +60,46 @@ func (s *Server) deleteBoard(w http.ResponseWriter, r *http.Request) {
 	render.Respond(w, r, nil)
 }
 
+func (s *Server) getUserBoards(w http.ResponseWriter, r *http.Request) {
+	log := logger.FromRequest(r)
+	user := r.Context().Value("User").(uuid.UUID)
+
+	boards, err := s.boards.GetUserBoards(r.Context(), user)
+	if err != nil {
+		log.Errorw("unable to get boards for this user", "err", err)
+		common.Throw(w, r, common.InternalServerError)
+		return
+	}
+	boardsWithDetails := make([]*dto.BoardWithDetails, len(boards))
+	for i, board := range boards {
+		board, _, sessions, cols, _, _, _, _, _, err := s.boards.FullBoard(r.Context(), board.ID)
+		participantNum := len(sessions)
+		if err != nil {
+			log.Errorw("unable to get full board details", "err", err)
```

Comment:
This error message also seems a bit odd, because the full board details aren't requested (only some details for overview), so why should this error occur